main/export.js

```diff
@@ -62,7 +62,7 @@ class Export {
           });
         }
         resolve();
-      } catch (error) {
+      } catch (_) {
         reject();
```

Comment:
Why not reject with the error here? 🤔

Reply:
I was going to ask @karaggeorge about that. I changed it to `_` to make it clear that the error is ignored.

Reply:
Yep, better for another PR perhaps

Reply:
Hmm, I don't remember. Probably should `reject(error)` but yeah we can leave that for a separate PR if you want